lib/basedriver/capabilities.js

```diff
@@ -96,7 +96,7 @@ function stripAppiumPrefixes (caps) {
 
   // If we found standard caps that were incorrectly prefixed, throw an exception (e.g.: don't accept 'appium:platformName', only accept just 'platformName')
   if (badPrefixedCaps.length > 0) {
-    throw new errors.InvalidArgumentError(`The capabilities ${JSON.stringify(badPrefixedCaps)} are standard capabilities and should not have the "appium:" prefix`);
+    log.warn(`The capabilities ${JSON.stringify(badPrefixedCaps)} are standard capabilities and do not require "appium:" prefix`);
```

Comment:
here is for standard capabilities.
So I think it should ignore capabilities which have `:` separator.

I mean `alwaysMatch: {'appium:platformName': 'Whatevz'}...` should be rejected, but `alwaysMatch: {'sauce:options': 'Whatevz'}...` should accept. But `alwaysMatch: {'sauce:platformName': 'Whatevz'}...` should be rejected.